drivers/display/Kconfig.stm32_ltdc

```diff
@@ -37,3 +39,23 @@ config STM32_LTDC_RGB565
 	  (2 bytes per pixel)
 
 endchoice
+
+config STM32_LTDC_FB_NUM
+	int "Frame buffer number"
+	default 1
+	range 0 2
```

Comment:
Non blocking: Maybe set range to 1 2, since 0 frame buffer config is not supported by the driver.

Reply:
0 is set for external user, like lvgl. when 0 is set, stm32_ltdc only support update the frame buffer address.